modules/list.js

```diff
@@ -0,0 +1,22 @@
+import Book from './book.js';
+
+export default class List {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('List')) || [];
+  }
+
+  add(title, author) {
+    const book = new Book(title, author);
+    this.list.push(book);
+    this.store();
+  }
+
+  remove(id) {
+    this.list = this.list.filter((book) => book.id !== id);
+    this.store();
+  }
+
+  store() {
+    localStorage.setItem('List', JSON.stringify(this.list));
+  }
```

Comment:
- [ ] One of the requirements for this project is to use the **_arrow syntax_** to create functions instead of the conventional way of creating functions. Remember that **_arrow syntax_** is one of the major introductions in _**ES6**_. Since you are practising **_ES6_**, it would be nice to use **_arrow syntax_**. Kindly convert your functions to use arrow syntax. 👍 😎 

#### Hint: You can leave the `constructor` method the way it is. 👍